app/experimenter/experiments/tests/test_tasks.py

```diff
@@ -373,3 +374,104 @@ def test_internal_only_does_not_add_comment(self):
         self.mock_bugzilla_requests_post.assert_not_called()
 
         self.assertEqual(Notification.objects.count(), 0)
+
+
+class TestUpdateExperimentStatus(
+    MockRequestMixin, MockNormandyMixin, TestCase
+):
+
+    def test_experiment_without_normandy_id(self):
+        ExperimentFactory.create_with_status(
+            target_status=Experiment.STATUS_ACCEPTED
+        )
+        tasks.update_experiment_status()
+        self.mock_normandy_requests_get.assert_not_called()
+
+    def test_accepted_experiment_becomes_live_if_normandy_enabled(self):
+        ExperimentFactory.create_with_status(
+            target_status=Experiment.STATUS_ACCEPTED, normandy_id=1234
+        )
+        tasks.update_experiment_status()
+        experiment = Experiment.objects.get(normandy_id=1234)
+        self.assertEqual(experiment.status, Experiment.STATUS_LIVE)
+        self.assertTrue(
+            experiment.changes.filter(
+                old_status=Experiment.STATUS_ACCEPTED,
```

Comment:
We should also check that the `changed_by.email` matches the email in the normandy data